pages/common/mocha.md

```diff
@@ -0,0 +1,15 @@
+# mocha
+
+> Execute Mocha JavaScript test runner
+
+- Run tests with default configuration, or as configured in `mocha.opts`
+
+`mocha`
+
+- Run tests that match a specific grep pattern
+
+`mocha --grep {{^regex$}}`
+
+- Run tests every time files change until you kill it
```

Comment:
How about something like "Run tests every time a file in the current directory changes", so that it gives more information? Or are the watched files configured through `mocha.opts` as well?